addons/payment/static/src/js/tour.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('payment.tour', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+    var tour = require('web_tour.tour');
+
+    var _t = core._t;
+
+    tour.register('payment_tour', {
+        'skip_enabled': true,
+    }, [{
+        trigger: ".oe_stat_button",
+        content: _t("Once your setup is ready, set your credentials for production mode and activate it."),
+        position: "left"
+    }, ]);
```

Comment:
What is the purpose of adding a tour with only one step ?

Reply:
I added it after the first review by @fgi-odoo 

from specs:
Add a blinking tip (from onboarding tours) on Production button when the acquirer is activated but not yet in production, to better emphasize on this mandatory step:
    Once your setup ready, set your credentials for production mode and activate it.

Reply:
Ok then.